client/app/modules/server-error/server-error.component.html

```diff
@@ -6,7 +6,7 @@ <h2>Sorry, server goes wrong</h2>
         </div>
 
         <div class="err-body">
-            <md-button ui-sref="home" class="md-raised md-primary">
+            <md-button ng-click="$ctrl.backToSystem()" class="md-raised md-primary">
```

Comment:
add disabled state for this button while sending a request

Reply:
fixed